homeassistant/components/soundtouch/services.yaml

```diff
@@ -0,0 +1,33 @@
+soundtouch_play_everywhere:
```

Comment:
This is not enough, since the services are registered under the `"media_player"` domain. The services need to be registered under the `"soundtouch"` domain for this change to work.

Reply:
Oh okay, so what would I have to do accomplish that? :)

Reply:
It may be enough to change here:
https://github.com/home-assistant/home-assistant/blob/071476343c10cdf8fe4a448082021eca3df7c8d7/homeassistant/components/soundtouch/media_player.py#L155

Reply:
If we change it we should make sure the service call still works and we update the breaking change paragraph to include the changes.

Reply:
> 
> 
> It may be enough to change here:
> 
> https://github.com/home-assistant/home-assistant/blob/071476343c10cdf8fe4a448082021eca3df7c8d7/homeassistant/components/soundtouch/media_player.py#L155

@MartinHjelmare I am sorry but I am still not sure what exactly needs to be changed there. Could you elaborate?

Reply:
On that line replace the media_player domain name with the soundtouch domain name.

As mentioned above, please test all changes when you think you're done.

Reply:
Okay thanks for the response. Howevever, I would not like to change the domain for all soundtouch services. 
@MartinHjelmare  What would I have to change to keep all soundtouch services under the media_player domain?

Reply:
Custom services should be registered under the integration domain.

Reply:
Hope e88695d709bad238e1f23a8cfeec9e23fbcdfab3 fixes this problem. 
Tests are noted and definitely mandatory though I am still struggling to set up a development environment on windows. I will test my suggested changes once I have successfully set it up. Hope you guys understand :)

Reply:
these instructions were helpful for me in windows 10: https://developers.home-assistant.io/docs/en/development_environment.html#developing-on-windows

Reply:
@dshokouhi thanks I am currently following them and hoping for the best.